2024-08-09 04:28:53,799 - distributed.nanny - INFO -         Start Nanny at: 'tcp://10.34.59.10:36735'
2024-08-09 04:28:53,803 - distributed.nanny - INFO -         Start Nanny at: 'tcp://10.34.59.10:39165'
2024-08-09 04:28:53,806 - distributed.nanny - INFO -         Start Nanny at: 'tcp://10.34.59.10:39338'
2024-08-09 04:28:53,806 - distributed.nanny - INFO -         Start Nanny at: 'tcp://10.34.59.10:36541'
2024-08-09 04:28:54,404 - distributed.diskutils - INFO - Found stale lock file and directory '/tmp/dask-scratch-space/worker-976s0iqk', purging
2024-08-09 04:28:54,404 - distributed.diskutils - INFO - Found stale lock file and directory '/tmp/dask-scratch-space/worker-4p3f93zh', purging
2024-08-09 04:28:54,405 - distributed.diskutils - INFO - Found stale lock file and directory '/tmp/dask-scratch-space/worker-fhjzieu3', purging
2024-08-09 04:28:54,405 - distributed.diskutils - INFO - Found stale lock file and directory '/tmp/dask-scratch-space/worker-ldqla4ym', purging
2024-08-09 04:28:54,927 - distributed.worker - INFO -       Start worker at:    tcp://10.34.59.10:39362
2024-08-09 04:28:54,928 - distributed.worker - INFO -       Start worker at:    tcp://10.34.59.10:33644
2024-08-09 04:28:54,928 - distributed.worker - INFO -          Listening to:    tcp://10.34.59.10:39362
2024-08-09 04:28:54,928 - distributed.worker - INFO -       Start worker at:    tcp://10.34.59.10:41865
2024-08-09 04:28:54,928 - distributed.worker - INFO -       Start worker at:    tcp://10.34.59.10:40129
2024-08-09 04:28:54,928 - distributed.worker - INFO -           Worker name:           SLURMCluster-6-3
2024-08-09 04:28:54,928 - distributed.worker - INFO -          Listening to:    tcp://10.34.59.10:33644
2024-08-09 04:28:54,928 - distributed.worker - INFO -          Listening to:    tcp://10.34.59.10:40129
2024-08-09 04:28:54,928 - distributed.worker - INFO -          Listening to:    tcp://10.34.59.10:41865
2024-08-09 04:28:54,928 - distributed.worker - INFO -          dashboard at:          10.34.59.10:41447
2024-08-09 04:28:54,928 - distributed.worker - INFO -           Worker name:           SLURMCluster-6-2
2024-08-09 04:28:54,928 - distributed.worker - INFO -           Worker name:           SLURMCluster-6-0
2024-08-09 04:28:54,928 - distributed.worker - INFO - Waiting to connect to:     tcp://10.34.59.1:39562
2024-08-09 04:28:54,928 - distributed.worker - INFO -           Worker name:           SLURMCluster-6-1
2024-08-09 04:28:54,928 - distributed.worker - INFO -          dashboard at:          10.34.59.10:37421
2024-08-09 04:28:54,928 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 04:28:54,928 - distributed.worker - INFO -          dashboard at:          10.34.59.10:39933
2024-08-09 04:28:54,928 - distributed.worker - INFO -          dashboard at:          10.34.59.10:44696
2024-08-09 04:28:54,928 - distributed.worker - INFO - Waiting to connect to:     tcp://10.34.59.1:39562
2024-08-09 04:28:54,928 - distributed.worker - INFO -               Threads:                          1
2024-08-09 04:28:54,928 - distributed.worker - INFO - Waiting to connect to:     tcp://10.34.59.1:39562
2024-08-09 04:28:54,928 - distributed.worker - INFO - Waiting to connect to:     tcp://10.34.59.1:39562
2024-08-09 04:28:54,928 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 04:28:54,928 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 04:28:54,928 - distributed.worker - INFO -                Memory:                   1.86 GiB
2024-08-09 04:28:54,928 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 04:28:54,928 - distributed.worker - INFO -               Threads:                          1
2024-08-09 04:28:54,928 - distributed.worker - INFO -               Threads:                          1
2024-08-09 04:28:54,928 - distributed.worker - INFO -       Local Directory: /tmp/dask-scratch-space/worker-98h9pt8w
2024-08-09 04:28:54,928 - distributed.worker - INFO -               Threads:                          1
2024-08-09 04:28:54,928 - distributed.worker - INFO -                Memory:                   1.86 GiB
2024-08-09 04:28:54,928 - distributed.worker - INFO -                Memory:                   1.86 GiB
2024-08-09 04:28:54,928 - distributed.worker - INFO -                Memory:                   1.86 GiB
2024-08-09 04:28:54,928 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 04:28:54,928 - distributed.worker - INFO -       Local Directory: /tmp/dask-scratch-space/worker-pqijs670
2024-08-09 04:28:54,928 - distributed.worker - INFO -       Local Directory: /tmp/dask-scratch-space/worker-yl6b6_4s
2024-08-09 04:28:54,928 - distributed.worker - INFO -       Local Directory: /tmp/dask-scratch-space/worker-f3aehcdm
2024-08-09 04:28:54,928 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 04:28:54,928 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 04:28:54,928 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 04:28:55,339 - distributed.worker - INFO - Starting Worker plugin shuffle
2024-08-09 04:28:55,340 - distributed.worker - INFO -         Registered to:     tcp://10.34.59.1:39562
2024-08-09 04:28:55,340 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 04:28:55,340 - distributed.worker - INFO - Starting Worker plugin shuffle
2024-08-09 04:28:55,340 - distributed.core - INFO - Starting established connection to tcp://10.34.59.1:39562
2024-08-09 04:28:55,340 - distributed.worker - INFO -         Registered to:     tcp://10.34.59.1:39562
2024-08-09 04:28:55,341 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 04:28:55,341 - distributed.core - INFO - Starting established connection to tcp://10.34.59.1:39562
2024-08-09 04:28:55,341 - distributed.worker - INFO - Starting Worker plugin shuffle
2024-08-09 04:28:55,342 - distributed.worker - INFO - Starting Worker plugin shuffle
2024-08-09 04:28:55,342 - distributed.worker - INFO -         Registered to:     tcp://10.34.59.1:39562
2024-08-09 04:28:55,342 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 04:28:55,342 - distributed.worker - INFO -         Registered to:     tcp://10.34.59.1:39562
2024-08-09 04:28:55,343 - distributed.core - INFO - Starting established connection to tcp://10.34.59.1:39562
2024-08-09 04:28:55,343 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 04:28:55,343 - distributed.core - INFO - Starting established connection to tcp://10.34.59.1:39562
slurmstepd: error: *** JOB 2329103 ON cl-node010 CANCELLED AT 2024-08-09T04:29:03 ***
2024-08-09 04:29:03,438 - distributed.worker - INFO - Stopping worker at tcp://10.34.59.10:40129. Reason: scheduler-close
2024-08-09 04:29:03,438 - distributed.worker - INFO - Stopping worker at tcp://10.34.59.10:41865. Reason: scheduler-close
2024-08-09 04:29:03,438 - distributed.worker - INFO - Stopping worker at tcp://10.34.59.10:39362. Reason: scheduler-close
2024-08-09 04:29:03,438 - distributed.worker - INFO - Stopping worker at tcp://10.34.59.10:33644. Reason: scheduler-close
2024-08-09 04:29:03,439 - distributed.batched - INFO - Batched Comm Closed <TCP (closed) Worker->Scheduler local=tcp://10.34.59.10:40510 remote=tcp://10.34.59.1:39562>
Traceback (most recent call last):
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/comm/tcp.py", line 297, in write
    raise StreamClosedError()
tornado.iostream.StreamClosedError: Stream is closed

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/batched.py", line 115, in _background_send
    nbytes = yield coro
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/tornado/gen.py", line 767, in run
    value = future.result()
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/comm/tcp.py", line 307, in write
    convert_stream_closed_error(self, e)
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/comm/tcp.py", line 142, in convert_stream_closed_error
    raise CommClosedError(f"in {obj}: {exc}") from exc
distributed.comm.core.CommClosedError: in <TCP (closed) Worker->Scheduler local=tcp://10.34.59.10:40510 remote=tcp://10.34.59.1:39562>: Stream is closed
2024-08-09 04:29:03,440 - distributed.batched - INFO - Batched Comm Closed <TCP (closed) Worker->Scheduler local=tcp://10.34.59.10:40511 remote=tcp://10.34.59.1:39562>
Traceback (most recent call last):
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/comm/tcp.py", line 297, in write
    raise StreamClosedError()
tornado.iostream.StreamClosedError: Stream is closed

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/batched.py", line 115, in _background_send
    nbytes = yield coro
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/tornado/gen.py", line 767, in run
    value = future.result()
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/comm/tcp.py", line 307, in write
    convert_stream_closed_error(self, e)
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/comm/tcp.py", line 142, in convert_stream_closed_error
    raise CommClosedError(f"in {obj}: {exc}") from exc
distributed.comm.core.CommClosedError: in <TCP (closed) Worker->Scheduler local=tcp://10.34.59.10:40511 remote=tcp://10.34.59.1:39562>: Stream is closed
2024-08-09 04:29:03,440 - distributed.batched - INFO - Batched Comm Closed <TCP (closed) Worker->Scheduler local=tcp://10.34.59.10:40506 remote=tcp://10.34.59.1:39562>
Traceback (most recent call last):
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/comm/tcp.py", line 297, in write
    raise StreamClosedError()
tornado.iostream.StreamClosedError: Stream is closed

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/batched.py", line 115, in _background_send
    nbytes = yield coro
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/tornado/gen.py", line 767, in run
    value = future.result()
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/comm/tcp.py", line 307, in write
    convert_stream_closed_error(self, e)
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/comm/tcp.py", line 142, in convert_stream_closed_error
    raise CommClosedError(f"in {obj}: {exc}") from exc
distributed.comm.core.CommClosedError: in <TCP (closed) Worker->Scheduler local=tcp://10.34.59.10:40506 remote=tcp://10.34.59.1:39562>: Stream is closed
2024-08-09 04:29:03,440 - distributed.batched - INFO - Batched Comm Closed <TCP (closed) Worker->Scheduler local=tcp://10.34.59.10:40508 remote=tcp://10.34.59.1:39562>
Traceback (most recent call last):
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/comm/tcp.py", line 297, in write
    raise StreamClosedError()
tornado.iostream.StreamClosedError: Stream is closed

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/batched.py", line 115, in _background_send
    nbytes = yield coro
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/tornado/gen.py", line 767, in run
    value = future.result()
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/comm/tcp.py", line 307, in write
    convert_stream_closed_error(self, e)
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/comm/tcp.py", line 142, in convert_stream_closed_error
    raise CommClosedError(f"in {obj}: {exc}") from exc
distributed.comm.core.CommClosedError: in <TCP (closed) Worker->Scheduler local=tcp://10.34.59.10:40508 remote=tcp://10.34.59.1:39562>: Stream is closed
2024-08-09 04:29:03,442 - distributed.nanny - INFO - Closing Nanny gracefully at 'tcp://10.34.59.10:39338'. Reason: scheduler-close
2024-08-09 04:29:03,443 - distributed.nanny - INFO - Closing Nanny gracefully at 'tcp://10.34.59.10:36541'. Reason: scheduler-close
2024-08-09 04:29:03,444 - distributed.nanny - INFO - Closing Nanny gracefully at 'tcp://10.34.59.10:39165'. Reason: scheduler-close
2024-08-09 04:29:03,444 - distributed.nanny - INFO - Closing Nanny gracefully at 'tcp://10.34.59.10:36735'. Reason: scheduler-close
2024-08-09 04:29:03,444 - distributed.core - INFO - Received 'close-stream' from tcp://10.34.59.1:39562; closing.
2024-08-09 04:29:03,444 - distributed.nanny - INFO - Worker closed
2024-08-09 04:29:03,445 - distributed.core - INFO - Received 'close-stream' from tcp://10.34.59.1:39562; closing.
2024-08-09 04:29:03,445 - distributed.nanny - INFO - Worker closed
2024-08-09 04:29:03,445 - distributed.core - INFO - Received 'close-stream' from tcp://10.34.59.1:39562; closing.
2024-08-09 04:29:03,445 - distributed.nanny - INFO - Worker closed
2024-08-09 04:29:03,446 - distributed.core - INFO - Received 'close-stream' from tcp://10.34.59.1:39562; closing.
2024-08-09 04:29:03,446 - distributed.nanny - INFO - Worker closed
